dipy/direction/bingham.py

```diff
@@ -540,100 +556,123 @@ def global_voxel_metric(bmetric, bfd):
 
 
 class BinghamMetrics:
-    """ Class for Bingham Metrics"""
+    r"""
+    Class for Bingham Metrics."""
 
     def __init__(self, model_params):
-        """Initialization of the Bingham Metrics Class
+        """ Initialization of the Bingham Metrics Class.
 
         Parameters
         ----------
-        model_params : ndarray (..., nl, 12)
-            ndarray containing the model parameters of Binghams fitted to ODFs
+        model_params : ndarray (..., nl, 16)
+            ndarray containing Bingham's model parameters fitted to ODFs
             in the following order:
             Maximum value of the Bingham function (f0, index 0);
             concentration parameters k1 and k2 (indexes 1 and 2);
-            elements of Bingham main direction (indexes 3-5);
-            elements of Bingham dispersion major axis (indexes 6-8);
-            and elements of Bingham dispersion minor axis (indexes 9-11).
+            elements of Bingham's main direction (indexes 3-5);
+            elements of Bingham's dispersion major axis (indexes 6-8);
+            elements of Bingham's dispersion minor axis (indexes 9-11);
+            peak directions (indexes 12-14);
+            peak indices  (index 15).
         """
         self.model_params = model_params
 
-        self.peak_dirs = model_params[..., 3:6]
+        self.bingham_peak_dirs = model_params[..., 3:6]
+        self.peak_dirs = model_params[..., 12:15]
         self.peak_values = model_params[..., 0]
+        self.peak_indices = model_params[..., 15]
```

Comment:
Waiting on these to be reverted before merging (A personal note just to remember where we are at).

Reply:
@CHrlS98 Will start working on this now. Just one more thing. Have you started working on the comments of @skoudoro?

Reply:
No. I want to avoid merge conflicts. I'll address them once I have merged your PR.

Reply:
Ah okay. I'll let that to you then. Will let you know when I have pushed.